Backend/DataAccessLayer/DataAccessLayer.csproj

```diff
@@ -75,13 +73,12 @@
     <Service Include="{508349B6-6B84-4DF5-91F0-309BEEBAD82D}" />
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="Migrations\201812040636053_create-table-user.resx">
-      <DependentUpon>201812040636053_create-table-user.cs</DependentUpon>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Migrations\201812040642468_create-table-session.resx">
-      <DependentUpon>201812040642468_create-table-session.cs</DependentUpon>
+    <EmbeddedResource Include="Migrations\201903062125096_initial-create.resx">
+      <DependentUpon>201903062125096_initial-create.cs</DependentUpon>
     </EmbeddedResource>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Folder Include="DTOs\" />
+  </ItemGroup>
```

Comment:
Why this change?

Reply:
I created a folder to put the LoginRequest class there but decided to move it into the model class